ts/services/ReportService.ts

```diff
@@ -62,6 +62,30 @@ class ReportService {
     }
   }
 
+  public async ec2recommendation(config, logger): Promise<void> {
+    logger.debug('\nListing EC2RecommendationData');
+    try {
+      const reports = await this.getReports(config, logger);
+      // TODO
+      const EC2RecommendationReport = reports.filter( (report) => report.type === 'EC2Cost' );
+      if (EC2RecommendationReport) {
+        const report = Array.isArray(EC2RecommendationReport) ? EC2RecommendationReport[0] : EC2RecommendationReport;
+        if (report) {
+          const elementsInScope = await this.getElementsInScope(config, logger, report);
+          if (elementsInScope.content) {
+            const elementFilter = elementsInScope.content.map( (x) => x.element_id );
+            const Recommendation = await this.getRecommendation(config, logger, report, elementFilter);
+            if (config.format === 'text') {
+              logger.info(Recommendation);
```

Comment:
I thought there was more custom formatting on the other report, turns out there's not. Logging the report directly is a good start so I think we can leave it like this.